pywinauto/element_info.py

```diff
@@ -166,6 +166,22 @@ def filter_with_depth(elements, root, depth):
         else:
             return elements
 
+    @staticmethod
+    def get_descendants_with_depth(elem, depth=None, **kwargs):
```

Comment:
Don't see much sense in `@staticmethod` as `elem` plays role of `self`. No problem to call `walk_the_tree(self, depth, **kwargs)`.

Reply:
Oops, I rewrited it.